# HG changeset patch
# Parent b83891a3279fe2c8e0eac95338ec716b3266b193
# User Frédéric Wang <fred.wang@free.fr>
Make reftest-analyzer.xhtml validate as XHTML5. r=dbaron

diff --git a/layout/tools/reftest/reftest-analyzer.xhtml b/layout/tools/reftest/reftest-analyzer.xhtml
--- a/layout/tools/reftest/reftest-analyzer.xhtml
+++ b/layout/tools/reftest/reftest-analyzer.xhtml
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!-- -*- Mode: HTML; tab-width: 4; indent-tabs-mode: nil; -*- -->
 <!-- vim: set shiftwidth=4 tabstop=4 autoindent noexpandtab: -->
 <!-- ***** BEGIN LICENSE BLOCK *****
    - Version: MPL 1.1/GPL 2.0/LGPL 2.1
    -
    - The contents of this file are subject to the Mozilla Public License Version
    - 1.1 (the "License"); you may not use this file except in compliance with
    - the License. You may obtain a copy of the License at
    - http://www.mozilla.org/MPL/
@@ -46,16 +47,17 @@ Features to add:
 * Add ability to load log via XMLHttpRequest (also triggered via URL param)
 * color the test list based on pass/fail and expected/unexpected/random/skip
 * ability to load multiple logs ?
 ** rename them by clicking on the name and editing
 ** turn the test list into a collapsing tree view
 ** move log loading into popup from viewer UI
 
 -->
+<!DOCTYPE html>
 <html lang="en-US" xml:lang="en-US" xmlns="http://www.w3.org/1999/xhtml">
 <head>
   <title>Reftest analyzer</title>
   <style type="text/css"><![CDATA[
 
   html, body { margin: 0; }
   html { padding: 0; }
   body { padding: 4px; }
@@ -510,17 +512,17 @@ function show_pixelinfo(x, y, pix1rgb, p
   </div>
   <div id="itemlist"></div>
   <div id="images" style="display:none">
     <form id="imgcontrols">
     <label><input type="radio" name="which" value="0" onchange="show_image(1)" checked="checked" />Image 1</label>
     <label><input type="radio" name="which" value="1" onchange="show_image(2)" />Image 2</label>
     <label><input type="checkbox" onchange="show_differences(this)" />Circle differences</label>
     </form>
-    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="800px" height="1000px" viewbox="0 0 800 1000" id="svg">
+    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="800px" height="1000px" viewBox="0 0 800 1000" id="svg">
       <defs>
         <!-- use sRGB to avoid loss of data -->
         <filter id="showDifferences" x="0%" y="0%" width="100%" height="100%"
                 style="color-interpolation-filters: sRGB">
           <feImage id="feimage1" result="img1" xlink:href="#image1" />
           <feImage id="feimage2" result="img2" xlink:href="#image2" />
           <!-- inv1 and inv2 are the images with RGB inverted -->
           <feComponentTransfer result="inv1" in="img1">
